### DISCUSSION TITLE
Heroku ValueError: BSI_CONFIG is not set

### DISCUSSION THREAD

**Discussion by raffiag** [ccrsxx/autobsi#17] [Q&A] [answered]
![Screenshot_197](https://user-images.githubusercontent.com/85484854/189867595-8ada81ec-dabf-41f0-8c29-3c9784db8f92.jpg)

**Comment by ccrsxx** [comment, marked as answer]
Ini ambil data dari `get_from_env` atau `get_from_heroku`. Klo misalnya pake dari `config.json` ganti dulu di `main.py`.

Pake settingan config dibawah:
```py
from src import get_from_config

setup = {
    'get': get_from_config,
    'mail': False,
    'verbose': False,
    'cloud': True,
}
```
